code/utility/palindrome/palindrome_check/palindrome_check.py

```diff
@@ -0,0 +1,26 @@
+def is_palindrome(arr):
```

Comment:
I remember already implementing this in the past: https://github.com/OpenGenus/cosmos/blob/master/code/string_algorithms/palindrome_checker/palindrome.py. The only thing that needs to be fixed is the recursive implementation.

Reply:
You could consider moving all the files there to palindrome_check.

Reply:
I remove my file, and move all palindrome from string_algos. to utiility